quran/quran.com-frontend-next
# [PWA] improve icons clarity and add svg support

### Summary
The PWA images look pixelated and scaled up from a lower resolution when being installed so I improved the clarity by exporting from an SVG while keeping the same size and adding SVG support to the icons for the PWA + improving the size with better compression.


### Test Plan
- Install the PWA app 
- Check the changed icons

### Screenshots

#### Android splash screen:

##### Before:
![Screenshot_20240407_073016_Chrome](https://github.com/quran/quran.com-frontend-next/assets/35064105/972527ff-f703-4334-b788-61727359b4a5)
 
##### After:
![Screenshot_20240407_072725_Chrome](https://github.com/quran/quran.com-frontend-next/assets/35064105/b9542313-797c-4c95-84e0-ac5b453ec264)


![image](https://github.com/quran/quran.com-frontend-next/assets/35064105/7333cc1c-c472-4da3-b5d9-3383f6d5fad6)
